packages/modular-scripts/src/esbuild-scripts/utils/openBrowser.ts

```diff
@@ -14,7 +14,7 @@ enum Actions {
   SCRIPT = 'Script',
 }
 
-const DEFAULT_BROWSER = process.env.BROWSER || OSX_CHROME;
+const DEFAULT_BROWSER = process.env.BROWSER || '';
```

Comment:
Comment here would be useful.

Reply:
TL;DR: fixed with amore readable type assertion

This is interesting because the typescript version always failed in CI (it always tried to open Chrome the OSX way), but we never saw that because we didn't start a view with esbuild before.
I took a look, and it seems that the `OSX_CHROME` default was put there erroneously to shut up the compiler that failed later when trying to start a Node headless "browser". In reality, if we try to start Node we are sure that the argument is a string (because we test it), but the compiler is not smart enough to understand it. Giving it a false-ish default value works, because the Node path is never false-ish, while the browser path tests for false-ishness anyway. But you're right that' incomprehensible, so I fixed it with a type assertion. The "right-est" way to fix this would be probably rewriting it to use a discriminated union, but that will make the js and ts versions diverge, which is another problem not in scope here. IMHO we should aim, in the future, to use the ts version everywhere, but that's for another effort.